app/src/org/commcare/utils/FileUtil.java

```diff
@@ -356,18 +356,17 @@ public static String getGlobalStringUri(String fileLocation) {
         return "file://" + fileLocation;
     }
 
-    public static void checkReferenceURI(Resource r, String URI, Vector<MissingMediaException> problems) throws InvalidReferenceException {
-        Reference mRef = ReferenceManager.instance().DeriveReference(URI);
+    public static Reference checkReferenceUri(String uri) throws InvalidReferenceException {

```

Comment:
this is causing the build to fail now due to method calls not having the corresponding change, you can ignore lint errors here.

Reply:
this will need to be reverted.